src/Microsoft.AspNet.OData/Microsoft.AspNet.OData.csproj

```diff
@@ -47,15 +47,24 @@
       <Private>True</Private>
     </Reference>
     <Reference Include="System" />
+    <Reference Include="System.Buffers, Version=4.0.3.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51, processorArchitecture=MSIL">
```

Comment:
Does the benefit of being able to use a `ReadOnlySpan` justify the cost of adding the 3 dependencies?

Reply:
I think so. Is there any performance cost with adding new dependencies?